### PR TITLE
Get notes and Get account

### PR DESCRIPTION
**Pull request by kfleury**
#### Description

Fixes #<issue number>

#### Changelog

- [BUGFIX]
- [ENHANCEMENT]
- [DOCS]
- ...

### CODE REVIEW

File: src/hooks/api/notes.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+export const getNotes = async (author_id: string) => {
+  const auth = useAuthContext()
+
+  const notes = await axios.get(`${API_BASE}/notes?author_id=${author_id}`, {
+    headers: {
+      'Authorization': `Bearer ${await auth.token()}`,
+    },
+  })
+  return notes.data
+}
```
Comment (dhbrojas) [inline src/hooks/api/notes.ts:26]: Wsh configure your editor to insert newlines at the end of files!

File: src/hooks/api/notes.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+export const getNotes = async (author_id: string) => {
+  const auth = useAuthContext()
+
+  const notes = await axios.get(`${API_BASE}/notes?author_id=${author_id}`, {
+    headers: {
+      'Authorization': `Bearer ${await auth.token()}`,
+    },
+  })
+  return notes.data
```
Comment (dhbrojas) [inline src/hooks/api/notes.ts:25]: Do you still need this?

Comment (balthi-tech) [inline src/hooks/api/notes.ts:25]: pas fou

**Review by balthi-tech — CHANGES_REQUESTED** [review]
a revoir dr drugs

**Review by dhbrojas — APPROVED** [review]

**Comment by edouard-sn** [comment]
https://static.wikia.nocookie.net/meme/images/5/57/Meme_Approved_Knuckles_but_he_stamped_too_hard/revision/latest/scale-to-width-down/340?cb=20200429125114

**Pull request merged**